src/pages/VeSDL/OnChainVote.tsx

```diff
@@ -61,24 +62,25 @@ export default function OnChainVote({
     () =>
       Object.keys(gauges)
         .filter((lpAddress) => !gauges[lpAddress]?.isKilled)
-        .map((lpAddress) => {
-          const gaugeName = gauges[lpAddress]?.gaugeName
-          const gaugeAddress = gauges[lpAddress]?.address
+        .reduce((acc, curr) => {
```

Comment:
instead of changing this to reduce, can you check 
```
if (!gauges[lpAddress]) return ____ (whatever the appropriate "empty" value is)
```
reduce isn't very functional

Reply:
Updated from reduce to using foreach